WeakAurasOptions/SubRegionOptions/Border.lua

```diff
@@ -0,0 +1,118 @@
+if not WeakAuras.IsCorrectVersion() then return end
+
+local SharedMedia = LibStub("LibSharedMedia-3.0");
+local L = WeakAuras.L;
+
+local screenWidth, screenHeight = math.ceil(GetScreenWidth() / 20) * 20, math.ceil(GetScreenHeight() / 20) * 20;
+
+local function DeleteSubRegion(data, index, regionType)
```

Comment:
These functions shouldn't be duplicated, we need to find a common place for them.

Also I'm adding duplicate in my PR.